src/components/main/index.tsx

```diff
@@ -1,47 +1,43 @@
-  },
-  {
-    id: 4, title: "Avengers: Infinity War", releaseDate: "2018-01-01", genres: ["Action", "Adventure"], image: "/img/avengers.png", rating: 7, movieUrl: "", runtime: 180, overview: "Avengers: Infinity War is a 2018 American superhero film based on the Marvel Comics superhero team the Avengers. Produced by Marvel Studios and distributed by Walt Disney Studios Motion Pictures, it is the sequel to The Avengers (2012) and Avengers: Age of Ultron (2015), and the 19th film in the Marvel Cinematic Universe (MCU). Directed by Anthony and Joe Russo and written by Christopher Markus and Stephen McFeely, the film features an ensemble cast including Robert Downey Jr., Chris Hemsworth, Mark Ruffalo, Chris Evans, Scarlett Johansson, Benedict Cumberbatch, Don Cheadle, Tom Holland, Chadwick Boseman, Paul Bettany, Elizabeth Olsen, Anthony Mackie, Sebastian Stan, Danai Gurira, Letitia Wright, Dave Bautista, Zoe Saldaña, Josh Brolin, and Chris Pratt. In the film, the Avengers and the Guardians of the Galaxy attempt to prevent Thanos from collecting the six all-powerful Infinity Stones as part of his quest to kill half of all life in the universe.",
-  },
-  {
-    id: 5, title: "Inception", releaseDate: "2010-01-01", genres: ["Action", "Adventure"], image: "/img/inception.png", rating: 6, movieUrl: "", runtime: 100, overview: "Inception is a 2010 science fiction action film[4][5][6] written and directed by Christopher Nolan, who also produced the film with Emma Thomas, his wife. The film stars Leonardo DiCaprio as a professional thief who steals information by infiltrating the subconscious of his targets. He is offered a chance to have his criminal history erased as payment for the implantation of another person's idea into a target's subconscious.[7] The ensemble cast includes Ken Watanabe, Joseph Gordon-Levitt, Marion Cotillard, Elliot Page,[a] Tom Hardy, Dileep Rao, Cillian Murphy, Tom Berenger, and Michael Caine.",
-  },
-  {
-    id: 6, title: "Reservoir Dogs", releaseDate: "1997-01-01", genres: ["Oscar Winning Movie"], image: "/img/reservoirDogs.png", rating: 5, movieUrl: "", runtime: 120, overview: "Reservoir Dogs is a 1992 American crime film written and directed by Quentin Tarantino in his feature-length debut. It stars Harvey Keitel, Tim Roth, Chris Penn, Steve Buscemi, Lawrence Tierney, Michael Madsen, Tarantino, and Edward Bunker as diamond thieves whose heist of a jewelry store goes terribly wrong. Kirk Baltz, Randy Brooks, and Steven Wright also play supporting roles. It incorporates many motifs that have become Tarantino's hallmarks: violent crime, pop culture references, profanity, and nonlinear storytelling.",
-  },
-];
+type State = {
+  movies: Movie[],
+  moviesQuantity: number,
```

Comment:
why do you need `moviesQuantity`? Can't you use just the length of `movies`?

Reply:
Because I thought that the number shown referred to the number of movies in the API, and not the number shown on the screen or page